torch/csrc/jit/passes/shape_analysis.cpp

```diff
@@ -1701,7 +1701,7 @@ class ShapePropagator {
       }
     } else if (
         node->matches(
-            "aten::embedding(Tensor weight, Tensor indices, int padding_idx, bool scale_grad_by_freq, bool sparse) -> Tensor")) {
+            "aten::embedding(Tensor weight, Tensor indices, int? padding_idx=None, bool scale_grad_by_freq=False, bool sparse=False, float? max_norm=None, float norm_type=2) -> Tensor")) {
```

Comment:
The jit test seems to still be failing.
Maybe changing this to detect `_embedding` would be simpler?